docs/user-guide/index.md

```diff
@@ -146,6 +146,48 @@ If you want to use custom parsers such as [@babel/eslint-parser](https://www.npm
   }
 ```
 
+Full example:
+
+::: code-group
+
+```json [Legacy Config]
```

Comment:
Thanks for the PR!
I think we'll avoid using the terms "Legacy Config" and "Flat Config" in our documentation.
Could you please change it to use `.eslintrc` and `eslint.config.js` instead?

Reply:
Sure thing! I saw the same naming convention elsewhere in the docs, I changed those too^^